platform/ios/PolyPodApp/PodApi/Network/Network.swift

```diff
@@ -38,25 +49,76 @@ class Network: NetworkProtocol {
             data, response, error in
             guard let response = response as? HTTPURLResponse,
                   error == nil else {
-                errorMessage = error?.localizedDescription ?? "Unknown error"
+                      errorMessage = error?.localizedDescription ?? "Unknown error"
+                      semaphore.signal()
+                      return
+                  }
+            
+            guard (200 ... 299) ~= response.statusCode else {
```

Comment:
Maybe define a constant Range? But again, probably better to just get fixed responses in the specs.